### PR TITLE
USID regex

### PR DESCRIPTION
**Pull request by dammy230**
extracting UUID numbers from Regex

### CODE REVIEW

**Review by elseagle — CHANGES_REQUESTED** [review]
Good work. Your PR name is confusing : I'm not sure we have any task for `USID`. Do you mean `UUID`?

Also, you were meant to pick just one task - you picked 2.

In addition; your file names have spaces in them and some are uppercase. You should name files with lowercase and join multiple words with an underscore e.g `api_using_file_name`.

Lastly what does `API USING FILE NAME.py` do?

File: API USING FILE NAME.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#After various API steps, integrate with python
+
+import gspread
+
+gc = gspread.service_account(filename=r"copy file path on pc")
```
[the remark this answers is not in the document]
Comment (dammy230) [reply API USING FILE NAME.py:5]: Noted.
thank you

File: API USING FILE NAME.py
[the remark this answers is not in the document]
Comment (elseagle) [reply API USING FILE NAME.py:1]: @dammy230 ?

File: collecting Kenya phone number.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+'''
+#search= re.compile(r"[+254]\d\d\d\d\d\d\d\d\d\d\d\d")
+#search= re.compile(r"[+254]\d{12}")
+search= re.compile(r"[+254]+\d{9}")
+matches=search.finditer(pattern)
+
+for match in matches:
+    print(match)
```
[the remark this answers is not in the document]
Comment (elseagle) [reply collecting Kenya phone number.py:12]: @dammy230 ?

**Pull request closed without merging**